mountpoint-s3/tests/fuse_tests/prefetch_test.rs

```diff
@@ -44,6 +42,7 @@ where
     let mut buf = vec![0u8; read_size];
     f.read_exact(&mut buf).expect("Should be able to read file to buf");
 
+    // changed the value of read_buf to distinguish it from previous data of the object.
     read_buf = vec![255u8; OBJECT_SIZE];
```

Comment:
```suggestion
    let new_data_buf = vec![255u8; OBJECT_SIZE];
```

Reply:
Also, the assert on line 69 could compare `dest_buf` with a slice from this buffer.